homeassistant/components/ruuvi_gateway/bluetooth.py

```diff
@@ -62,6 +64,7 @@ def _async_handle_new_data(self) -> None:
                 manufacturer_data=anno.manufacturer_data,
                 tx_power=anno.tx_power,
                 details={},
+                advertisement_monotonic_time=monotonic_now - data_age_seconds,
```

Comment:
I'm not sure if we actually need to fudge this with `data_age_seconds`..?

Reply:
It's used to compare if an advertisement is stale.  If the gateway goes offline without notice and another scanner can see the device having the newer timestamp would cause the manager to delay switching to the available scanner